use newly created badge-plugin

bakito opened the pull request:
set java level 8

use badge-plugin v1.0

ikedam commented:
-1 for auto-formatter.
It makes difficult to review and to track the change history with `git blame`.
I know groovy-postbuild is really badly formatted, but I prefer preserving as is.

bakito commented:
@ikedam Sorry for that
I've reverted the re-formatting and updated the pull request.
I've also updated some dependencies with version that are supported by version 2.60 https://updates.jenkins.io/2.60/latest/. I hope that is ok with you.

I'm not yet familiar on how to choose the correct plugin versions.

bakito commented:
@ikedam Thank you for the detailed Review. I'll make the modifications later this day.

bakito commented on pom.xml:
```diff
@@ -23,34 +23,23 @@
+    <java.level>8</java.level>
+    <badge-plugin.version>1.0</badge-plugin.version>
+    <workflow-step-api-plugin.version>2.14</workflow-step-api-plugin.version>
+    <workflow-cps-plugin.version>2.45</workflow-cps-plugin.version>
```
Is it ok for you, if I updated the jenkins.version to 2.60.3 or even newer?

ikedam replied:
No problem. Please go ahead.
Also badge-plugin shlould target 2.60.3+.

bakito replied:
correct, I was waiting with the release, if you had any changes.

ikedam commented:
Pushed this change as "extract-badge" branch for the subsequent pull request.
https://github.com/jenkinsci/groovy-postbuild-plugin/tree/extract-badge

ikedam commented:
@bakito 
The badge-plugin integration is released as groovy-postbuild-2.4.
It will be available in the update center in a day.
